packages/react/src/themes/teams/components/Status/statusVariables.ts

```diff
@@ -11,11 +12,24 @@ export interface StatusVariables {
   errorTextColor: string
   defaultBackgroundColor: string
   defaultTextColor: string
+
+  backgroundRed: string
+  backgroundOrange: string
+  backgroundYellow: string
+  backgroundGreen: string
+  backgroundBlue: string
+  backgroundViolet: string
+
+  smallest: string
+  smaller: string
+  small: string
+  medium: string
+  large: string
+  larger: string
+  largest: string
```

Comment:
These used to be computed in the style function, based on the `size` prop.  Now that props aren't exposed in the style functions, we should pre-calculate all values and hoist them to variables.

Styles only receive variables at this point.